garden-backend-service/src/api/routes/entrypoints.py

```diff
@@ -49,6 +52,37 @@ async def get_entrypoint_by_doi(
     return entrypoint
 
 
+@router.get("", status_code=status.HTTP_200_OK)
+async def get_entrypoints(
+    db: AsyncSession = Depends(get_db_session),
+    *,
+    doi: list[str] | None = Query(None),
+    tags: list[str] | None = Query(None),
+    authors: list[str] | None = Query(None),
+    owner_uuid: UUID | None = Query(None),
+    draft: bool | None = Query(None),
+    year: str | None = Query(None),
+    limit: int = Query(50),
+) -> list[EntrypointMetadataResponse]:
```

Comment:
Do we want to set a max limit? Relevant [docs](https://fastapi.tiangolo.com/reference/parameters/#fastapi.Query)
```suggestion
    limit: int = Query(50, le=100),
) -> list[EntrypointMetadataResponse]:
```

Reply:
ah yeah that's a good call. 100 seems sane, easy to change later if needed